--- src/main/META-INF/MANIFEST.MF.orig	2010-11-02 07:16:18.000000000 +0200
+++ src/main/META-INF/MANIFEST.MF	2010-12-06 14:25:21.242038734 +0200
@@ -1,16 +1,19 @@
-Manifest-Version: 1.0
-Implementation-Title: ${title}
-Implementation-URL: http://www.h2database.com
-Implementation-Version: ${version}
-Build-Jdk: ${buildJdk}
-Created-By: ${createdBy}
-${mainClassTag}
-Bundle-Activator: org.h2.util.DbDriverActivator
-Bundle-ManifestVersion: 2
-Bundle-Name: H2 Database Engine
-Bundle-SymbolicName: org.h2
-Bundle-Vendor: H2 Group
-Bundle-Version: ${version}
-DynamicImport-Package: *
-Export-Package: org.h2,org.h2.api,org.h2.fulltext,org.h2.jdbcx,org.h2.tools,org.h2.util
-Premain-Class: org.h2.util.Profiler
+Manifest-Version: 1.0
+Ant-Version: Apache Ant 1.8.1
+Created-By: ${createdBy}
+Implementation-Title: H2 Database Engine
+Implementation-URL: http://www.h2database.com
+Implementation-Version: ${version}
+Build-Jdk: 1.6
+Bundle-Activator: org.h2.util.DbDriverActivator
+Bundle-ManifestVersion: 2
+Bundle-Name: H2 Database Engine
+Bundle-SymbolicName: org.h2
+Bundle-Vendor: H2 Group
+Bundle-Version: 1.2.145
+DynamicImport-Package: *
+Export-Package: org.h2,org.h2.api,org.h2.fulltext,org.h2.jdbcx,org.h2.
+ tools,org.h2.util
+Premain-Class: org.h2.util.Profiler
+Main-Class: org.h2.tools.Console
+
